content/blog/2020/12/2020-12-04-gsod-project-report.adoc

```diff
@@ -0,0 +1,114 @@
+
+* Improve the user experience of Jenkins on Kubernetes users by giving them a one-stop-shop for information on Jenkins on Kubernetes.
+* Make it easy to track, update and maintain information on Jenkins on Kubernetes
+* Reference the existing community documentation for Jenkins on K8s (plugins and tools/integrations).
+* How to guides, tutorials and explanations of concepts and techniques in Jenkins on Kubernetes.
+* Just-In-Time documentation which means that rather than documenting every feature comprehensively, we produce and release documentation in bits but continuously based on popular questions, feedback and area of interests gathered from the community and users.
+
+
+## Community Bonding: Planning the solution
+
+Find below an outline of my activities during the community bonding phase:
+
+* Setting up communication channels: meetings, mailings, chats: My mentors and I agreed on the right time and channel for communication due to time difference.
+We agreed to meet twice weekly, on Mondays and Thursdays at 7:00 PM GMT +1 and use Jenkins documentation link:https://gitter.im/jenkinsci/docs[gitter] channel for other communications.
+* Contacting Stakeholders and onboarding contributors: The project was  announced on social media and different Jenkins channels.
+I wrote a link:/blog/2020/09/25/document-jenkins-on-kubernetes-introduction/[blog post] to announce the project and created a link:/sigs/docs/gsod/2020/projects/document-jenkins-on-kubernetes/[project page] on Jenkins.io.
+* Knowledge transfer: I and my mentors planned knowledge sharing sessions and fixed tentative dates based on the availability of the trainers.
+My mentors also shared useful resources to help me prepare for the project.
+* Getting permissions: I and my mentors agreed I didn’t need any special permissions from the beginning, however, this topic was left open for discussion if the need arose later on in the project.
+* Pre-planning of the project: I refined my goals and set expectations with my mentor and also learned more about the community(Jenkins).
+I also had to ensure that the proposed documentation structure I drafted was in line with the goals of the organization so my mentors vetted it and we finalized on the proposed sections that I was supposed to work on.
+
+
+## Documentation Development Phase
+
+
+### Knowledge Sharing Sessions
+
+During the development phase, my mentors hosted two knowledge sharing sessions:
+
+* link:https://youtu.be/BkIiGXDCEGA[Katakoda and Helm] by link:https://twitter.com/markyjackson5[Marky Jackson], you can find the slides for this session link:https://docs.google.com/presentation/d/1LD7btYQaSiI3R8226OIAI6EIUqRZHTyjefVMdF4ydSg/edit?usp=sharing[here].
+* link:https://youtu.be/9WIGVLBIfNM[Helm] by link:https://twitter.com/torsten_walter[Torsten Walter], you can find the slides for this session link:https://docs.google.com/presentation/d/1RS8PwlR_FzxYypBlwtp4LcZls8hr3dG_4KJ65U00Xlo/edit#slide=id.gc6f80d1ff_0_0[here].
```

Comment:
@zaycodes if you'd like to embed the video player into the blog post, this is how you might do it:

```suggestion
#### Katakoda and Helm by link:https://twitter.com/markyjackson5[Marky Jackson]

See the link:https://docs.google.com/presentation/d/1LD7btYQaSiI3R8226OIAI6EIUqRZHTyjefVMdF4ydSg/edit?usp=sharing[slides]

video::BkIiGXDCEGA[youtube,width=800,height=420]

#### Helm by link:https://twitter.com/torsten_walter[Torsten Walter]

See the link:https://docs.google.com/presentation/d/1RS8PwlR_FzxYypBlwtp4LcZls8hr3dG_4KJ65U00Xlo/edit#slide=id.gc6f80d1ff_0_0[slides]

video::9WIGVLBIfNM[youtube,width=800,height=420]
```

With the video players, the blog post looks like this:

![screencapture-localhost-4242-blog-2020-12-04-gsod-project-report-2020-12-03-18_49_32](https://user-images.githubusercontent.com/156685/101112245-19432a80-359a-11eb-9091-33e1a9cacbf6.png)

Without them, it looks like this:

![screencapture-localhost-4242-blog-2020-12-04-gsod-project-report-2020-12-03-18_42_41](https://user-images.githubusercontent.com/156685/101112232-15170d00-359a-11eb-9cab-6618a2f11e41.png)


Either layout is fine with me.  We'll merge this Friday, Dec 4, 2020 with which ever layout you prefer.

Reply:
Thanks Mark. The post looks great with the video players. Will definitely use the layout with them